setup.py

```diff
@@ -357,10 +355,9 @@ def run(self):
     },
 
     # Disallow 3.9.0 because of https://github.com/python/cpython/pull/22670
-    python_requires='>=3.7, !=3.9.0',  # noqa: E501
+    python_requires='>=3.9, !=3.9.0',  # noqa: E501
```

Comment:
This expression can be simplified to 
```suggestion
    python_requires='>3.9.0',  # noqa: E501
```

```python
>>> from packaging.version import Version
>>> Version('3.9.1') > Version('3.9.0')
True
>>> Version('3.9.0') > Version('3.9.0')
False
```